js/Source/Delegators/Delegator.Ajax.js

```diff
@@ -84,6 +84,28 @@ name: Delegator.Ajax
         onFailure: function(e){
           if (api.get('errorRedirectURL')){
             window.location.href = api.get('errorRedirectURL');
+          } else if (api.get('failureTriggers')){
+            var delegators = api.getAs(Object, 'failureTriggers');
+            var delegator = api.getDelegator();
+            if (delegators){
+
+              // when the user mouses over/out, loop over the triggers
+              Object.each(delegators, function(config, trigger){
```

Comment:
your naming is confusing; you loop over `delegators` with each one being called a `trigger`. pick a side! i think the array should probably be called `triggers`.